common/app/com/commercetools/sunrise/shoppingcart/CartBeanFactory.java

```diff
@@ -16,12 +17,27 @@ public CartBean create(final Cart cart) {
         return bean;
     }
 
+    public CartBean createWithEmptyCart() {
```

Comment:
in CartDetailPageContent the parameter Cart is nullable, so why there are two different styles?